valetudo-map-card.js

```diff
@@ -760,6 +800,7 @@ class ValetudoMapCard extends HTMLElement {
       const wallColor = this.calculateColor(homeAssistant, this._config.wall_color, '--valetudo-map-wall-color', '--accent-color');
       const currentlyCleanedZoneColor = this.calculateColor(homeAssistant, this._config.currently_cleaned_zone_color, '--valetudo-currently_cleaned_zone_color', '--secondary-text-color');
       const noGoAreaColor = this.calculateColor(homeAssistant, this._config.no_go_area_color, '--valetudo-no-go-area-color', '--accent-color');
+      const noMopAreaColor = this.calculateColor(homeAssistant, this._config.no_mop_area_color, '--valetudo-no-mop-area-color', '--accent-color');
```

Comment:
Actually, "no mop" should probably use another colour. Right now it'll use the same colour as no go and that seems like it would be confusing.

Reply:
I know, but I had no idea what the default could be as there's no secondary accent-color.
Maybe secondary-text-color looks, I'll test how that looks.

Reply:
Is secondary-text-color ok? At the top is the small accent-color no_go_area, lower is the bigger secondary-text-color no_mop_area.
<img width="256" alt="Bildschirmfoto 2021-11-25 um 23 02 22" src="https://user-images.githubusercontent.com/8668637/143503388-11233b33-8b3a-40ea-86b0-89b0fbab25bc.png">

Reply:
Yeah seems perfect, people can tell the difference this way :)